aries-site/src/layouts/content/Example/Example.js

```diff
@@ -226,19 +229,26 @@ export const Example = ({
                     controls={showResponsiveControls}
                     onSetScreen={setScreen}
                     screen={screen}
+                    fullscreen={fullscreen}
+                    setFullscreen={setFullscreen}
                   />
                 )}
                 <Button
-                  title="Leave full screen"
+                  tip="Leave Fullscreen"
                   icon={<Contract />}
                   onClick={() => {
-                    setShowLayer(false);
+                    setFullscreen(false);
                     setScreen(screens.laptop);
                   }}
                   hoverIndicator
                 />
               </Box>
               <Box
+                animation={{
```

Comment:
This animation felt a little slow to me -- I was wondering if the example wasn't loading -- can we reduce the time / delay a little?

Reply:
Done.

Reply:
looks nice